### ISSUE TITLE
[Bug][a11y] Reader mode "Appearance" control menu - Light mode button size

### ISSUE THREAD

**Issue by eliserichards** [mozilla-mobile/fenix#18710]
## Steps to reproduce 
1. Have the Accessibility scanner installed and opened.
2. Go to a site with a lot of text content (e.g.wikipedia).
3. Enable "Reader view" from the navigation bar.
4. From the three-dot menu, tap on the "Appearance" menu.
5. Scan the "Appearance" control menu.

### Expected behavior
No suggestions are made.

### Actual behavior

Suggestions made for the **"Light"** button:

* Touch target
[754,1668][859,1773]
This item's size is 40dp x 40dp. Consider making this touch target 48dp wide and 48dp high or larger. A parent container may be handling touch events for this item. If selecting the larger container performs the same action as selecting this item, consider defining this item as not clickable. If a different action is performed, consider increasing the size of this item.

<img src="https://user-images.githubusercontent.com/40694158/98212539-6a84dd80-1f4c-11eb-804f-1327cf9ce92e.png" width="300" height="550" />

### Device information
* Android device: Google Pixel (Android 10)
* Fenix version: 11/3 Nightly build

**Comment by brngylni** [comment]
Can I work on it?

**Comment by eliserichards** [comment]
Hi @brngylni! Are you an Outreachy applicant? If so, feel free to work on this 👍

**Comment by brngylni** [comment]
> 
> 
> Hi @brngylni! Are you an Outreachy applicant? If so, feel free to work on this 👍

Yes, I am an Outreachy applicant. Ok then, I am starting to work. Thanks.

**Comment by brngylni** [comment]
It wouldn't be so bad if I could get some extra detail 
@eliserichards

**Comment by brngylni** [comment]
Hey, I think this is beyond me. You'd better assign this to someone else. I am sorry.
@eliserichards

**Comment by sanyud24** [comment]
As @brngylni left this issue . May I start working on this one ?

**Comment by eliserichards** [comment]
See https://github.com/mozilla-mobile/android-components/blob/1d1753abba4d29ad3e91c16d0c751db7cdbd639d/components/feature/readerview/src/main/java/mozilla/components/feature/readerview/ReaderViewFeature.kt#L78 in the [Android Components](https://github.com/mozilla-mobile/android-components) repo

**Comment by sanyud24** [comment]
Ohkay , thankyou 👍

**Comment by sanyud24** [comment]
may you review this PR @eliserichards
